0algo_id

```diff
@@ -0,0 +1,70 @@
+
+if [[ $COIN == FTC || $COIN == ORB || $COIN == PXC || $COIN == VIVO || $COIN == TZC || $COIN == NICE_NEOSCRYPT || $COIN == DSR || $COIN == CRC || $COIN == GBX || $COIN == MPH_NEOSCRYPT || $COIN == ZPOOL_NEOSCRYPT ]]
+then
+  ALGO=NEOSCRYPT
+elif [[ $COIN == ETH || $COIN == ETC || $COIN == EXP || $COIN == UBQ || $COIN == CLO || $COIN == MUSIC || $COIN == SOIL || $COIN == MPH_ETHASH || $COIN == PIRL || $COIN == NICE_ETHASH || $COIN == ETP ]]
+then
+  ALGO=ETHASH
+elif [[ $COIN == BTG || $COIN == ZEC || $COIN == ZEN || $COIN == HUSH || $COIN == ZCL || $COIN == BTCP || $COIN == KMD || $COIN == ZPOOL_EQUIHASH || $COIN == MPH_EQUIHASH || $COIN == NICE_EQUIHASH || $COIN == VOTE   || $COIN == BTCP  || $COIN == BCI ]]
+then
+  ALGO=EQUIHASH
+elif [[ $COIN == SIGT || $COIN == ZPOOL_SKUNK || $COIN == ALTCOM || $COIN == NICE_SKUNKHASH ]]
+then
+  ALGO=SKUNK
+elif [[ $COIN == MONA || $COIN == VTC || $COIN == NICE_LYRA2REV2 || $COIN == XVG || $COIN == MPH_LYRA2V2 || $COIN == ZPOOL_LYRA2V2 ]]
+then
+  ALGO=LYRA2V2
+elif [[ $COIN == KRB || $COIN == ETN || $COIN == NICE_CRYPTONIGHT ]]
+then
+  ALGO=CRYPTONIGHT
+elif [[ $COIN == ZCOIN || $COIN == MPH_LYRA2Z ]]
+then
+  ALGO=LYRA2Z
+elif [[ $COIN == BSD ]]
+then
+  ALGO=XEVAN
+elif [[ $COIN == RVN ]]
+then
+  ALGO=X16R
+elif [[ $COIN == BTX ]]
+then
+  ALGO=BITCORE
+elif [[ $COIN == SIB  || $COIN == NICE_X11GHOST ]]
+then
+  ALGO=SIB
+elif [[ $COIN == MONERO || $COIN == MPH_CRYPTONIGHTV7 || $COIN == NICE_CRYPTONIGHTV7 ]]
+then
+  ALGO=CRYPTONIGHTV7
+elif [[ $COIN == XMR || $COIN == LUX || $COIN == FLM ]]
+then
+  ALGO=PHI
+elif [[ $COIN == PASC || $COIN == PASL ]]
+then
+  ALGO=PASCAL
+elif [[ $COIN == LBC || $COIN == ZPOOL_LBRY ]]
+then
+  ALGO=LBRY
+elif [[ $COIN == DMD ]]
+then
+  ALGO=DMD-GR
+elif [[ $COIN == GRS || $COIN == MPH_GRS ]]
+then
+  ALGO=GRS
+elif [[ $COIN == XMY ]]
+then
+  ALGO=MYR-GR
```

Comment:
As @sizzlephizzle reminded, hyphen will break the variable, MYR-GR, and DMD-GR should be changed to underscore